triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -63,12 +63,18 @@ final case class CompletionMsg(c: Completion) extends TriggerMsg
 final case class TransactionMsg(t: Transaction) extends TriggerMsg
 final case class HeartbeatMsg() extends TriggerMsg
 
-final case class TypedExpr(expr: Expr, ty: TypeConApp)
+final case class TriggerDefinition(
+    id: Identifier,
+    ty: TypeConApp,
+    version: Trigger.Version,
+    lowLevel: Trigger.Level,
+    expr: Expr,
```

Comment:
`Trigger`'s `expr` was a `TypedExpr`, but this is just an `Expr`. Is it implied that the type is (a function of) `ty` above?

Reply:
Yep.